.github/workflows/python-dev.yml

```diff
@@ -1,11 +1,3 @@
-# This file is purposely frozen(does not run). DO NOT DELETE IT
```

Comment:
I think it would be better to leave the comment in, and just comment out the ``if: false`` below.

Reply:
Likewise. Even better to clarify

1. "Freeze" if our dependencies don't support the latest python dev version
2. "Unfreeze" if our dependencies support it, but the python dev version hasn't been officially released

Reply:
@fangchenli Do you have any objections here?

Reply:
> @fangchenli Do you have any objections here?

No, I agree with you and @mroeschke